tests/data/diff_pys_riotfile.py

```diff
@@ -0,0 +1,14 @@
+from riot import Venv
+
+venv = Venv(
+    venvs=[
+        Venv(
+            name="test",
+            command="exit 0",
+            pys=["2.7", "3.5", "3.6", "3.7", "3.8", "3.9"],
```

Comment:
can we make this unsorted to verify that it does get sorted?

Reply:
Thanks for the good catch!